_data/wai-evaluation-tools-list/submissions/c2512100-04d4-11f0-879c-311f258cbb73.json

```diff
@@ -0,0 +1,60 @@
+{
+  "a11yloc": "",
+  "actrules": "",
+  "assists": [
+    "Generating reports of evaluation results",
+    "Providing step-by-step evaluation guidance"
+  ],
+  "automated": [
+    "Component",
+    "Single page/screen",
+    "Sample of pages/screens",
+    "Entire website / software / product",
+    "Restricted or password protected pages"
+  ],
+  "comments": "",
+  "contact": "karthik@browserstack.com",
+  "desktop": [
+    "Android",
+    "iOS"
+  ],
+  "features": "Automate accessibility testing for mobile apps with just a single flag change. Run tests on real devices, catch violations early, and integrate seamlessly into your CI/CD pipeline. No maintenance needed—DOM updates are tracked automatically. Get detailed reports, monitor trends, and ensure your app stays accessible with every release.",
+  "form_name": "submission",
+  "form_version": "1",
+  "guideline": [
+    "WCAG 2.2",
+    "WCAG 2.1",
+    "WCAG 2.0"
```

Comment:
From the [documentation](https://www.browserstack.com/docs/app-accessibility/automated-tests/configuration-options), it seems you cannot target WCAG 2.0 specifically. @josehilera Would you ask for clarification from the vendor?